docs/build-docs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+rm -rf build/
+
+# Move theming elements into the docs folder
+cp -R pytket-docs-theming/_static .
+cp -R pytket-docs-theming/quantinuum-sphinx .
+cp pytket-docs-theming/conf.py .
+
+# Get the name of the project
+EXTENSION_NAME="$(basename "$(dirname $(pwd))")"
+
+# Correct github link in navbar
+sed -i '' 's#CQCL/tket#CQCL/'$EXTENSION_NAME'#' _static/nav-config.js
+
+# Build the docs. Ensure we have the correct project title.
+sphinx-build -b html -D html_title="$EXTENSION_NAME" . build
```

Comment:
It looks like the docs build is failing locally due to a missing `index.rst` file.

If you rename the `intro.txt` file to `index.rst` this should fix it. You get the syntax highlighting too as well :)

Also for consistency with the other extensions we can add the pypi and github issue links to the bottom of `index.rst` as well

FOR PYTKET-QIR WE HAVE
```
.. toctree::
   :caption: Useful links

   Issue tracker <https://github.com/CQCL/pytket-qir/issues>
   PyPi <https://pypi.org/project/pytket-qir/>
```